frontend/src/components/regions/UserActions/Presentation.jsx

```diff
@@ -4,8 +4,11 @@ import Grid from "@material-ui/core/Grid";
 import Loading from "../../widgets/Loading";
 import { makeStyles } from "@material-ui/core/styles";
 import DayLog from "./DayLog";
+
+import ActivityDashboardBarGraph from "./ActivityDashboardBarGraph";
```

Comment:
Don't import it here for now, Any changes made to this file can also be removed. We should only consider the storybook